example/.gitignore

```diff
@@ -1,2 +1,4 @@
 wetzel_reisebegleiter_1901_0021*.alto
 wetzel_reisebegleiter_1901_0021*.hocr
+wetzel_reisebegleiter_1901_0021*.page
+/out
```

Comment:
Git complains about a missing newline at end of file.

Reply:
Compare how much scrutiny this PR gets vs https://github.com/UB-Mannheim/ocr-fileformat/pull/98. But okay, I'll update.

Reply:
(FYI I have an even longer waiting PR #92...)

Reply:
PR #92 now has conflicts which must be fixed before merging. :-(

Reply:
No, not anymore:-)